File: UL/CppApp/System/Object.cpp
#include "stdafx.h"
#include "System\Object.h"
#include "System\Type.h"
#include "System\Boolean.h"
#include "System\String.h"
#include "Object_ExtCpp.h"
System::Boolean System::Object::op_Inequality(Ref<System::Object>  a,Ref<System::Object>  b)
{
	return System::Boolean::op_LogicNot((System::Object::op_Equality(a,b)));
}

File: UL/Debug/Lua/System/Object.lua
require "System"
System.Object = class('System.Object')
function System.Object:GetType()
end
function System.Object:Equals_System_Object(v)
end
function System.Object:ToString()
end
function System.Object.ReferenceEquals_System_Object_System_Object(a,b)
end
function System.Object.op_Equality(a,b)
end
function System.Object.op_Inequality(a,b)
	do
		return System.Boolean.op_LogicNot((System.Object.op_Equality(a,b)));
	end
end
require "Object_ExtHeader"
